editoast/openapi.yaml

```diff
@@ -3063,6 +3064,16 @@ components:
           type: string
         ch:
           type: string
+        track_sections:
+          type: object
+          required:
+            - track
+            - position
+          properties:
+            track:
+              type: string
+            position:
+              type: integer
```

Comment:
Shouldn't it be 
```suggestion
        track_sections:
          type: array
          items:
            required:
              - track
              - position
            properties:
              track:
                type: string
              position:
                type: integer
```

Reply:
Thanks! I'm dying for #3625 to be done...